api/config.py

```diff
@@ -52,27 +52,28 @@
     },
 }
 
-__config = copy.deepcopy(DEFAULT_CONFIG)
+def apply_env_variables(config):
+    # Overwrite default config values with SCITRAN env variables if available
+    for outer_key, scoped_config in config.iteritems():
+        if outer_key in ['core', 'site', 'auth', 'persistent']:
```

Comment:
Before we ignored anything in db config object on startup and overwrote it with env vars and default values. This change is setting the order of precedence to be env > db > default on startup.